CitiesSkylinesMods/TMPE
# 730 cleanup loading custom path manager

- Fixed master Debug build broken.

fixes #730 
- encapsulated code for loading/unloading CustomPathManager.
- TMPE can be removed mid-game (without the need to replace it with anther TMPE).
- bugfix: unloading TMPE replaces back the stock path find.
- removed redundant(no profile uses it) / stale PF_DIJKSTRA (it does not even build)

#### Test passed:
1. Load Game with TMPE
2. apply TMPE lane connections to force cars to change
3. observe that cars change lanes
4. delete TMPE
5. observe that cars no longer change lanes.
6. build TMPE again (hot swap)
7. observe that cars change lanes again.

 https://ci.appveyor.com/api/projects/krzychu124/tmpe/artifacts/TMPE.zip?branch=730-cleanup-loading-CustomPathManager
 
 blocks #1076 #1077
